src/extensions/renderer/canvas/drawing-nodes.js

```diff
@@ -155,12 +152,21 @@ CRp.drawNode = function( context, node, shiftToOriginWithBb, drawLabel ){
   //
   // bg image
 
-  if( urlDefined ){
-    if( image.complete ){
-      this.drawInscribedImage( context, image, node );
+  var prevBging = _p.backgrounding;
+  var totalCompleted = 0;
+
+  for( var i = 0; i < numImages; i++ ){
+    if( ( urlDefined[i] ) && image[i].complete ){
+      totalCompleted++;
+      this.drawInscribedImage( context, image, node, i );
```

Comment:
You should be passing in `image[i]`.  The function doesn't need an array of images.